source/element-types.md

```diff
@@ -1,4 +1,232 @@
 (c:ele.types)=
 # Element Types
+          
+
+Undulator           Undulator.
+Wiggler             Wiggler.
```

Comment:
Did you intend undulator or wiggler as a separate type heading here?  If so, following the template of the other element types, I would add a short description followed by "Under construction..."  If not, I would delete these lines for now.

Reply:
This was cut-and-paste garbage that I have removed.